metricbeat/module/elasticsearch/stats/_meta/data.json

```diff
@@ -0,0 +1,38 @@
+{
+    "@timestamp": "2016-05-23T08:05:34.853Z",
+    "beat": {
+        "hostname": "host.example.com",
+        "name": "host.example.com"
+    },
+    "elasticsearch": {
+        "stats": {
+            "docs": {
+                "count": 4,
+                "deleted": 0
+            },
+            "segments": {
+                "count": 4,
+                "memory": {
+                    "bytes": 6647
+                }
+            },
+            "shards": {
+                "failed": 0,
+                "successful": 11,
+                "total": 22
+            },
```

Comment:
I think docs, segments, shards all look fine. I'd like to have an indices section here as well, though, with a total count (or maybe closed and total)? Whatever you think makes sense on that.